conda/config.py

```diff
@@ -92,3 +92,8 @@ def get_rc_path():
 pkgs_dirs = context.pkgs_dirs
 default_prefix = context.default_prefix
 subdir = context.subdir
+
+# put back because of conda build
+default_python = '%d.%d' % sys.version_info[:2]
```

Comment:
use context here, like the lines just above

    default_python = context. default_python
    binstar_upload = context.binstar_upload